CMakeLists.txt

```diff
@@ -1,11 +1,31 @@
 # Generated by `boostdep --cmake sort`
 # Copyright 2020 Peter Dimov
+# Copyright 2025 Nigel Stewart
 # Distributed under the Boost Software License, Version 1.0.
 # https://www.boost.org/LICENSE_1_0.txt
 
-cmake_minimum_required(VERSION 3.5...3.16)
+message(STATUS "Using cmake version ${CMAKE_VERSION}")
 
-project(boost_sort VERSION "${BOOST_SUPERPROJECT_VERSION}" LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.10...3.16)
+
+if(BOOST_SUPERPROJECT_VERSION)
+  project(boost_sort VERSION "${BOOST_SUPERPROJECT_VERSION}" LANGUAGES CXX)
+else()
+  project(boost_sort LANGUAGES CXX)
+endif()
+
+# Test coverage in stand-alone mode requires boost dependencies
+
```

Comment:
Maybe remove this empty line to not disconnect it from the code block it applies

```suggestion
```